core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -37,7 +37,7 @@ import { ListSchema, UnicodeSchema } from 'services/schema-default-value.service
 import {
   TRANSLATION_DATA_FORMAT_SET_OF_NORMALIZED_STRING,
   TRANSLATION_DATA_FORMAT_SET_OF_UNICODE_STRING
-} from 'domain/exploration/WrittenTranslationObjectFactory';
+} from 'domain/exploration/writtentranslation-object.model';
```

Comment:
```suggestion
} from 'domain/exploration/written-translation.model';
```

Missing hyphen? Also "object" seems redundant here. Are we able to drop that?

Same comment applies for core/templates/pages/contributor-dashboard-page/services/translate-text.service.ts.